ORBIT/manager.py

```diff
@@ -1438,38 +1509,206 @@ def overnight_capex(self):
+            num_turbines = self.config["plant"]["num_turbines"]
+            rating = self.config["turbine"]["turbine_rating"]
+            procurement_contingency = procurement_contingency_per_kW * num_turbines * rating * 1000
+            
+        except:
+            procurement_contingency_factor =  self.project_params.get("procurement_contingency_factor", 0.0575)
+            procurement_contingency = (self.turbine_capex + self.bos_capex + self.project_capex - self.installation_capex) *\
+                                    procurement_contingency_factor
+
+        return procurement_contingency
+
+    @property
+    def installation_contingency_capex(self):
+        """
+        Returns the installation contingency capital cost of the project.
+        Methodology from ORCA model, default values used in 2022 Cost of Wind Energy Review.
+        """
+
+        try:
+            installation_contingency_per_kW = self.config["project_parameters"]["installation_contingency"]
+            num_turbines = self.config["plant"]["num_turbines"]
+            rating = self.config["turbine"]["turbine_rating"]
+            installation_contingency = installation_contingency_per_kW * num_turbines * rating * 1000
+            
+        except:
+            installation_contingency_factor =  self.project_params.get("installation_contingency_factor", 0.345)
+            installation_contingency = self.installation_capex * installation_contingency_factor
+
+        return installation_contingency
+
+    @property
+    def construction_financing_factor(self):
+        """
+        Returns the construction finaning factor of the project.
+        Methodology from ORCA model, default values used in 2022 Cost of Wind Energy Review,
+        except the spend schedule, which is sourced from collaborations with industry.
+        """
+
+        try:
+            spend_schedule = self.config["project_parameters"]["spend_schedule"]
+        except:
+            spend_schedule =  self.project_params.get("spend_schedule", {0: 0.25,
+                                                                         1: 0.25,
+                                                                         2: 0.3,
+                                                                         3: 0.1,
+                                                                         4: 0.1,
+                                                                         5: 0.0
+                                                                         })
+            
+        try:
+            tax_rate = self.config["project_parameters"]["tax_rate"]
+        except:
+            tax_rate =  self.project_params.get("tax_rate", 0.26)
+
+        try:
+            interest_during_construction = self.config["project_parameters"]["interest_during_construction"]
+        except:
+            interest_during_construction =  self.project_params.get("interest_during_construction", 0.044)
```

Comment:
same as above.

Reply:
It works without the try/except. Thanks for the suggestion.